### DISCUSSION TITLE
Viết báo cáo tuần 10

### DISCUSSION THREAD

**Discussion by baonhu1304** [baonhu1304/Lop-10A2-nhom-1#23] [General]
Đợt này xài canva báo cáo cho đẹp nha mn. Ai thích mẫu nào, thấy phù hợp thì nhắn vô đây giúp tui nghen

**Comment by letu25082006** [comment]
oki lun nheee

**Comment by thaopham2501** [comment]
t có thiết kế một mẫu rồi á :))) dành cho bài 7
còn bài 8 chưa làm :v

**Reply by thaopham2501** [reply]
mà cũng không hẳn là thiết kế :() https://www.canva.com/design/DAEvz_yJ5vU/share/preview?token=JbhiJ859h4eslsaavdCj-w&role=EDITOR&utm_content=DAEvz_yJ5vU&utm_campaign=designshare&utm_medium=link&utm_source=sharebutton

**Reply by thaopham2501** [reply]
@baonhu1304 :>>>>

**Reply by baonhu1304** [reply]
Oke dzô xem liền.

**Reply by thaopham2501** [reply]
ok hong :v

**Reply by baonhu1304** [reply]
gất oke lun

**Reply by thaopham2501** [reply]
ổ kê, đang làm cho bài 8, còn giờ m chèn bài 7 vô trang đó đi :?

**Reply by baonhu1304** [reply]
ok ok đợi xí t đang làm câu 2

**Reply by thaopham2501** [reply]
@baonhu1304 có giữ file câu hỏi không, gửi với

**Reply by baonhu1304** [reply]
T đăng lên tab code nha

**Reply by thaopham2501** [reply]
okiii'

**Reply by thaopham2501** [reply]
@baonhu1304 cỡ chữ có phải theo cáo template kia hong dzãy :((( làm trên canva mà theo cai kia thì nhỏ quá

**Reply by baonhu1304** [reply]
> @baonhu1304 cỡ chữ có phải theo cáo template kia hong dzãy :((( làm trên canva mà theo cai kia thì nhỏ quá
hõi thầy ii

**Reply by thaopham2501** [reply]
> > @baonhu1304 cỡ chữ có phải theo cáo template kia hong dzãy :((( làm trên canva mà theo cai kia thì nhỏ quá
> > hõi thầy ii

m hỏi hay t hỏi :))
